src/Symfony/Component/JsonPath/Test/JsonPathAssertionsTrait.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath\Test;
+
+use Symfony\Component\JsonPath\JsonCrawler;
+use Symfony\Component\JsonPath\JsonPath;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ *
+ * @psalm-requires-extend \PHPUnit\Framework\Assert
```

Comment:
```suggestion
 * @psalm-requires-extends \PHPUnit\Framework\Assert
```

Reply:
Also `require` doesn't take an `s` actually. Thank you!